src/models/tools/image/image-content.ts

```diff
@@ -0,0 +1,44 @@
+import { types, Instance } from "mobx-state-tree";
+
+export const kImageToolID = "Image";
+
+const NumberOrString = types.union(types.number, types.string);
+
+export const ImageContentModel = types
+  .model("ImageTool", {
+    type: types.optional(types.literal(kImageToolID), kImageToolID),
+    url: types.maybe(types.string),
+    width: types.maybe(NumberOrString),
+    height: types.maybe(NumberOrString),
```

Comment:
I'm inclined to think that width/height would be better placed in the layout object rather than the content object. Is there a reason to have separate dimensions for the tool and the image displayed by the tool? If both notions of dimensions exist than how are they synchronized?